config/environments/production.rb

```diff
@@ -100,4 +100,10 @@
   # config.active_record.database_selector = { delay: 2.seconds }
   # config.active_record.database_resolver = ActiveRecord::Middleware::DatabaseSelector::Resolver
   # config.active_record.database_resolver_context = ActiveRecord::Middleware::DatabaseSelector::Resolver::Session
+
+  config.action_controller.asset_host = "exercism-assets-staging.s3.eu-west-2.amazonaws.com"
 end
+
+Rails.application.routes.default_url_options = {
+  host: "http://exercism.lol"
+}
```

Comment:
This will be our staging domain - at least for a while. If the nameservers ever propgate!